### PR TITLE
fix(#151): Update react-base16-styling to latest

### PR DESCRIPTION
**Pull request by tannermares**
### Why this was needed
The fork of `react-base16-styling` being used currently is throwing an error in React Native 0.64. (see #151). I noticed the upstream repo had been moved into [`redux-devtools`](https://github.com/reduxjs/redux-devtools/tree/master/packages/react-base16-styling) and has had some work done to it recently. Let's update to the maintained dependency and update the API's that changed!

### How I did it
- `yarn add react-base16-styling`
- Shamelessly copied from [`react-json-tree`](https://github.com/reduxjs/redux-devtools/blob/master/packages/react-json-tree/src/index.tsx) to use the new `invertTheme` API and some helper method extractions. 
~- *Side note: We should get prettier in here, I didn't want to impose too much so I didn't add it.*~
- Added `prettier` and `eslint` packages to dev dependencies
- Added `prettierrc.js` and `eslintrc.js` files with sane defaults
  - I would normally turn semi's off, but I'm not sure on your preference
  - I also added a `prettier-sync` script to `package.json`. I would run this after this PR to get the whole repo pretty.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -63,41 +79,61 @@ class JSONTree extends React.Component {
...
     invertTheme: true,
     sortObjectKeys: true,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = getStateFromProps(props);
+  }
+
+  UNSAFE_componentWillReceiveProps(nextProps) {
```
Comment (Dean177) [inline src/index.js]: It looks like this  `UNSAFE_componentWillReceiveProps` and the constructor could be replaced with a [getDerivedStateFromProps](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops)

**Review by Dean177 — CHANGES_REQUESTED** [review]
Thanks a bunch @tannermares, if we can remove the deprecated lifecycle that would be great

**Comment by Dean177** [comment]
Also be sure to add yourself as a contributor to the package.json

**Comment by Dean177** [comment]
Totally agree on bringing in prettier. I think this project was created before it existed!

File: src/index.js (hunk 1 above)
Comment (tannermares) [reply src/index.js]:
```
Warning: `JSONTree` uses `getDerivedStateFromProps` but its initial state is undefined. This is not recommended. Instead, define the initial state by assigning an object to `this.state` in the constructor of `JSONTree`. This ensures that `getDerivedStateFromProps` arguments have a consistent shape.
```
I think we should keep the constructor initial state setting.

**Review by Dean177 — APPROVED** [review]

**Comment by Dean177** [comment]
Fixes #152

**Pull request merged**